test/Atmos/EDMF/stable_bl_edmf_implicit.jl

```diff
@@ -0,0 +1,180 @@
+using ClimateMachine
+ClimateMachine.init(;
+    parse_clargs = true,
+    output_dir = get(ENV, "CLIMATEMACHINE_SETTINGS_OUTPUT_DIR", "output"),
+    fix_rng_seed = true,
+)
+using ClimateMachine.SingleStackUtils
+using ClimateMachine.Checkpoint
+using ClimateMachine.SystemSolvers
+using ClimateMachine.ODESolvers
+using ClimateMachine.BalanceLaws: vars_state
+const clima_dir = dirname(dirname(pathof(ClimateMachine)));
+
+include(joinpath(clima_dir, "experiments", "AtmosLES", "stable_bl_model.jl"))
+include("edmf_model.jl")
+include("edmf_kernels.jl")
+include("stable_bl_edmf.jl")
```

Comment:
This includes a lot more than what we want, including `ClimateMachine.init(...)`, and a `main` definition. If we can't put the stable bl `init_state_prognostic!` into the stable bl model, then we should probably split it out into a separate file and include it where needed.

Reply:
@yairchn can do this